mopidy_local/commands.py

```diff
@@ -221,17 +221,17 @@ def _scan_metadata(
                     logger.warning(
                         f"Failed scanning {file_uri}: No audio found in file"
                     )
-                elif result.duration is None:
-                    logger.warning(
-                        f"Failed scanning {file_uri}: "
-                        "No duration information found in file"
-                    )
-                elif result.duration < MIN_DURATION_MS:
+                elif result.duration and result.duration < MIN_DURATION_MS:
```

Comment:
Just in case, this change will now let duration `0` pass and skip through all checks because it's not None and it's falsey